plugins/modules/ecs_service.py

```diff
@@ -535,7 +583,8 @@
 
 DEPLOYMENT_CONFIGURATION_TYPE_MAP = {
     'maximum_percent': 'int',
-    'minimum_healthy_percent': 'int'
+    'minimum_healthy_percent': 'int',
+    'deployment_circuit_breaker': 'dict'
```

Comment:
```suggestion
    'deployment_circuit_breaker': 'dict',
```

Reply:
I'm a fan of leaving the dangling comma on the end here, it means that future updates don't need to touch the other line